tests/cases/compiler/extendedUnicodePlaneIdentifiers.ts

```diff
@@ -2,3 +2,11 @@
 const 𝑚 = 4;
 const 𝑀 = 5;
 console.log(𝑀 + 𝑚); // 9
+
+// lower 8 bits look like 'a'
+const ၡ = 6;

```

Comment:
does this test really examine `isIdentifierText`? from what I can see it's only used to validate `compilerOptions.reactNamespace` and used for printing symbol names in `getNameOfSymbolAsWritten -> getNameOfSymbolFromNameType`. Other uses are in services for several completions.

Reply:
`ဒ` wasn't going to parse correctly without this change, since it's lower 8 bits aren't a valid character on it's own. `ၡ` wasn't really needed in hindsight, since `𝑀` already covers the case when the late 8 bits are a valid character.